packages/uniforms-bootstrap3/src/DateField.tsx

```diff
@@ -63,7 +66,7 @@ function Date({
       readOnly={readOnly}
       ref={inputRef}
       type={dateType}
-      value={dateFormat(value) ?? ''}
+      value={dateFormat(value, type) ?? ''}
```

Comment:
This could be simplified to:
```ts
      type={type === 'date' ? type : 'datetime-local'}
      value={value?.toISOString().slice(0, type === 'date' ? -14 : -8) ?? ''}
```

Then we signalize that it's the same logic. If needed, we can have a constant for `type === 'date'` (but I don't think we do).

Of course, this applies to all themes.

Reply:
Do you think this check makes sense if we specify the "type" type as `'date' | 'datetime-local'`?

```ts
type === 'date' ? type : 'datetime-local'
```

Reply:
Well, I don't see a different option...? Or maybe I misunderstood something.

Reply:
If `type` is typed as `'date' | 'datetime-local'` then we can just pass `type` directly instead of doing it via the ternary operator:

```diff
- type={type === 'date' ? type : 'datetime-local'}
+ type={type}
```

Reply:
Answering this and the one above: I'm not entirely sure, but I have a weird feeling that it's there for a reason... If you'll check it (locally) that the `type` never has an odd value (like, `String` for example), then I'm fine with that.